### ISSUE TITLE
support: xacros use deprecated url for ns declarations

### ISSUE THREAD

**Issue by gavanderhoorn** [ros-industrial/fanuc#239]
As per subject.

Example:

https://github.com/ros-industrial/fanuc/blob/21d16189c3640c7f161a56504edec4db26eb7735/fanuc_m10ia_support/urdf/m10ia.xacro#L2

This should be `http://wiki.ros.org/xacro`.

**Comment by gavanderhoorn** [comment]
When using these xacro macros with other macros that do have the correct url in the ns declaration, the following warning is shown by `xacro`:

```
inconsistent namespace redefinitions for xmlns:xacro:
 old: http://www.ros.org/wiki/xacro
 new: http://wiki.ros.org/xacro (/home/user/catkin_ws/src/fanuc/fanuc_m10ia_support/urdf/m10ia_macro.xacro)
```

**Comment by gavanderhoorn** [comment]
Fixed in #240.
